### DISCUSSION TITLE
Is it possible to use a formatter defined in xml inside the project?

### DISCUSSION THREAD

**Discussion by idelice** [mfussenegger/nvim-jdtls#569] [General]
I have a google-formatter.xml inside a spring boot project under resources that has defined rules and works fine in intellij, but it's not applying those rules when formatting inside neovim using nvim-jdtls.

here's my config that I've tried but i'm unsure if it's correctly setup:
```
      settings = {
        java = {
          format = {
            settings = {
              profile = "GoogleStyle",
              url = "/Users/me/projects/my-project/src/main/resources/checkstyle-google.xml",
            },
          },
        },
      },
```

**Comment by eruizc-dev** [comment]
As far as I know the eclipse language server does not support integration with any formatters / linters: https://github.com/eclipse-jdtls/eclipse.jdt.ls/issues/713.

What you can do instead is use [this plugin](https://www.vim.org/scripts/script.php?script_id=448) or with some effort you should be able to configure [EFM language server](https://github.com/mattn/efm-langserver) to work with Checkstyle.
